scripts/z-dist.celx

```diff
@@ -20,7 +20,7 @@ end
 
 function rgb2hex(r,g,b)
    -- Converts color code from RGB to Hex
-   local hex = string.format("#%.2x%.2x%.2x", r,g,b)
+   local hex = string.format("#%.2x%.2x%.2x", math.floor(r), math.floor(g), math.floor(b))
```

Comment:
Lua 5.3 is stricter in number conversions.

`Error: [string "scripts/z-dist.celx"]:23: bad argument #3 to 'format' (number has no integer representation)`